portal/client-services/org.wso2.is.portal.user.client.api/src/main/java/org/wso2/is/portal/user/client/api/micro/IdentityStoreClientMicroService.java

```diff
@@ -34,7 +37,6 @@
  * This is the micro service implementation for identity store client services.
  */
 public class IdentityStoreClientMicroService implements Microservice {
```

Comment:
As Generate Password is not a submit we needed to handle it using an ajax call. So had to use micro service.